contracts/src/PBHEntryPointImplV1.sol

```diff
@@ -251,19 +251,15 @@ contract PBHEntryPointImplV1 is IPBHEntryPoint, WorldIDImpl, ReentrancyGuardTran
         nonReentrant
         returns (IMulticall3.Result[] memory returnData)
     {
+        if (gasleft() > pbhGasLimit) {
+            revert GasLimitExceeded(gasleft());
```

Comment:
The `GasLimitExceeded` error is defined as `error GasLimitExceeded(uint256 gasLimit);`, however `gasLeft()` is emitted. We should update this to emit the current gas limit (or update the event and emit `gasLeft() - gasLimit`).